src/fragments/start/getting-started/android/setup.mdx

```diff
@@ -52,6 +52,9 @@ Amplify for Android is distributed as an Apache Maven package. In this section,
    dependencies {
        implementation 'com.amplifyframework:aws-api:ANDROID_VERSION'
        implementation 'com.amplifyframework:aws-datastore:ANDROID_VERSION'
+       def lifecycle_version = "2.4.0"
```

Comment:
Do we need to fix customers to this particular lifecycle_version? if not, we can leave it as variable similar as `ANDROID_VERSION` above.

Reply:
I went ahead and committed @salihgueler suggested change, I don't think there's a variable for lifecycle_version similar to ANDROID_VERSION